tests/Feature/Repository/UserTastes/UserTastesTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Tests\Feature\Repository\UserTastesTest;
+
```

Comment:
Empty line.

Reply:
I don't really understand what's wrong with this new line? New line after php tag, new line between namespace and use statements